telos-frontend/src/components/calendar/JournalCalendar.jsx

```diff
@@ -1,162 +1,153 @@
+    startDate: new Date('2021-03-21T20:00:00.000Z'),
+    endDate: new Date('2021-03-21T20:30:00.000Z'),
   },
-  progress: {
-    position: 'absolute',
-    width: '100%',
-    bottom: 0,
-    left: 0,
+  {
+    priorityId: 2,
+    typeId: 2,
+    title: 'SOFTENG 754',
+    startDate: new Date('2021-03-21T21:00:00.000Z'),
+    endDate: new Date('2021-03-21T23:00:00.000Z'),
   },
-};
-
-const theme = createMuiTheme({
-  palette: {
-    primary: purple,
+  {
+    priorityId: 2,
+    typeId: 2,
+    title: 'SOFTENG 701',
+    startDate: new Date('2021-03-21T23:00:00.000Z'),
+    endDate: new Date('2021-03-22T01:00:00.000Z'),
   },
-});
-
-const ToolbarWithLoading = withStyles(styles, { name: 'Toolbar' })(
-  ({ children, classes, ...restProps }) => (
-    <div className={classes.toolbarRoot}>
-      {/* eslint-disable-next-line react/jsx-props-no-spreading */}
-      <Toolbar.Root {...restProps}>{children}</Toolbar.Root>
-      <LinearProgress className={classes.progress} />
-    </div>
-  )
-);
-
-const usaTime = (date) =>
-  new Date(date).toLocaleString('en-US', { timeZone: 'America/Los_Angeles' });
+  {
+    priorityId: 1,
+    typeId: 1,
+    title: 'Lunch',
+    startDate: new Date('2021-03-22T01:00:00.000Z'),
+    endDate: new Date('2021-03-22T02:30:00.000Z'),
+  },
+  {
+    priorityId: 1,
+    typeId: 2,
+    title: '701 Meeting',
+    startDate: new Date('2021-03-22T03:30:00.000Z'),
+    endDate: new Date('2021-03-22T04:30:00.000Z'),
+  },
+  {
+    priorityId: 1,
+    typeId: 2,
+    title: '754 Meeting',
+    startDate: new Date('2021-03-22T04:30:00.000Z'),
+    endDate: new Date('2021-03-22T05:30:00.000Z'),
+  },
+  {
+    priorityId: 2,
+    typeId: 2,
+    title: 'Work',
+    startDate: new Date('2021-03-22T06:00:00.000Z'),
+    endDate: new Date('2021-03-22T08:00:00.000Z'),
+  },
+];
 
-const mapAppointmentData = (appointment) => ({
-  id: appointment.id,
-  startDate: usaTime(appointment.start.dateTime),
-  endDate: usaTime(appointment.end.dateTime),
-  title: appointment.summary,
-});
+export const priorityData = [
+  {
+    text: 'Low Priority',
+    id: 1,
+    color: '#D6ADFF',
+  },
+  {
+    text: 'High Priority',
+    id: 2,
+    color: '#7A00F4',
+  },
+];
 
-const initialState = {
-  data: [],
-  loading: false,
-  currentDate: '2017-05-23',
-  currentViewName: 'Day',
-};
+export const typeData = [
+  {
+    text: 'Home',
+    id: 1,
+    color: '#ADADFF',
+  },
+  {
+    text: 'University',
+    id: 2,
+    color: '#0000de',
+  },
+];
+export default class Calendar extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: calendarData,
+      resources: [
+        {
+          fieldName: 'priorityId',
+          title: 'Priority',
+          instances: priorityData,
+          allowMultiple: false,
+        },
+        {
+          fieldName: 'typeId',
+          title: 'Type',
+          instances: typeData,
+          allowMultiple: false,
+        },
+      ],
+    };
```

Comment:
I'd like this to be refactored in the future to keep it tidy, but I'm happy to approve for demo purposes.